### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: CHANGELOG.rst
Comment (nicoddemus) [inline CHANGELOG.rst:13]: #201

Comment (Zac-HD) [reply CHANGELOG.rst:13]: I think non-ascii source code already worked fine; the problem was that we insert the grandparent-directory into `sys.path` and if _that path_ contained non-ascii characters then we'd crash.

Comment (Zac-HD) [inline CHANGELOG.rst:13]:
```suggestion
* Fixed error when loading source code files from a path containing non-ascii characters.
```

**Review by Zac-HD — APPROVED** [review]

File: CHANGELOG.rst
Comment (nicoddemus) [reply CHANGELOG.rst:13]: Ahh thanks!

**Comment by nicoddemus** [comment]
The branch needs to be pushed to `upstream`, not my fork. 🤦‍♂️ 

Superseded by #203.

**Comment by nicoddemus** [comment]
Waiting for deployment approval from @pytest-dev/execnet-admin. 👍

**Pull request closed without merging**
